### PR TITLE
DOCS-1317: Document `get_geometries` for all components

### PR DESCRIPTION
**Pull request by sguequierre**
* Comment out where not yet supported in Go (was confused about implementation but realized only select components currently subclass the interface `resource.Shaped` which provides that method.) Motion team intends to implement everywhere/open ticket so leaving commented.

### CODE REVIEW

File: docs/components/arm/_index.md [hunk 1]
```diff
@@ -516,6 +516,64 @@ logger.Info(is_moving)
 {{% /tab %}}
 {{< /tabs >}}
 
+### GetGeometries
+
+Get all the geometries associated with the arm in its current configuration, in the [frame](/services/frame-system/) of the arm.
+The [motion](/services/motion/) and [navigation](/services/navigation/) services use the relative position of inherent geometries to configured geometries representing obstacles for obstacle detection and avoidance while motion planning.
```
Comment (sguequierre) [inline docs/components/arm/_index.md:522]: @biotinker thoughts on accuracy of phrasing? Can change

File: docs/components/board/_index.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ For configuration information, click on the model name:
 | ----- | ----------- |
 | [`pi`](pi/) | [Raspberry Pi 4](https://www.raspberrypi.com/products/raspberry-pi-4-model-b/), [Raspberry Pi 3](https://www.raspberrypi.com/products/raspberry-pi-3-model-b/) or [Raspberry Pi Zero 2 W](https://www.raspberrypi.com/products/raspberry-pi-zero-2-w/) |
 | [`jetson`](jetson/) | [NVIDIA Jetson AGX Orin](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-orin/), [NVIDIA Jetson Orin Nano](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-orin/), [NVIDIA Jetson Xavier NX](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-xavier-nx/), [NVIDIA Jetson Nano](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-nano/) |
-| [`upboard`](upboard/) | An Intel-based board like the [UP4000](https://github.com/up-board/up-community/wiki/Pinout_UP4000) |
+| [`upboard`](upboard/) | An Intel-boardd board like the [UP4000](https://github.com/up-board/up-community/wiki/Pinout_UP4000) |
```
Comment (npentrel) [inline docs/components/board/_index.md:51]: I think you did one search and replace too many

**Review by npentrel — APPROVED** [review]
LGTM % search and replace error and SME approval

File: docs/components/board/_index.md (hunk 2 above)
Comment (sguequierre) [reply docs/components/board/_index.md:51]: Ahh thank you!! Fixing

Comment (sguequierre) [inline docs/components/board/_index.md:51]:
```suggestion
| [`upboard`](upboard/) | An Intel-based board like the [UP4000](https://github.com/up-board/up-community/wiki/Pinout_UP4000) |
```

File: docs/components/arm/_index.md (hunk 1 above)
Comment (biotinker) [reply docs/components/arm/_index.md:522]: > obstacle detection and avoidance

"obstacle detection" is done elsewhere and is unrelated (for now) to `GetGeometries`. I would call this "collision detection and obstacle avoidance".

Other than that this sounds good.

File: docs/components/arm/_index.md [hunk 3]
```diff
@@ -516,6 +516,64 @@ logger.Info(is_moving)
...
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/arm/client/index.html#viam.components.arm.client.ArmClient.get_geometries).
+
+```python {class="line-numbers linkable-line-numbers"}
+my_arm = Arm.from_robot(robot=robot, name="my_arm")
+
+geometries = await my_arm.get_geometries()
+
+if geometries:
+    # Get the center of the first geometry
+    print(f"Pose of the first geometry's center point: {geometries[0].center}")
```
Comment (biotinker) [inline docs/components/arm/_index.md:545]: I think this should just be "center", or combined to "centerpoint". To me having the space between "center" and "point" combined with "Pose" implies things about points and orientations that aren't always true.

File: docs/components/arm/_index.md (hunk 1 above)
Comment (sguequierre) [reply docs/components/arm/_index.md]: Fixed, thank you! @biotinker

**Pull request merged**